tests/flashcore/test_review_processor.py

```
@@ -0,0 +1,442 @@
+"""
+Tests for the ReviewProcessor class.
+
+The ReviewProcessor consolidates the core review submission logic that was
+previously duplicated between ReviewSessionManager and _review_all_logic.py.
+"""
+
+import pytest
+from datetime import datetime, timezone, date
+from uuid import uuid4, UUID
```

Ruff (0.11.9)

`uuid.UUID` imported but unused

Remove unused import: `uuid.UUID`

(F401)